wdl/RecoverCCSRemainingReads.wdl

```diff
@@ -32,7 +32,7 @@ task RecoverCCSRemainingReads {
         boot_disk_gb:       10,
         preemptible_tries:  1,
         max_retries:        0,
-        docker:             "kgarimella/lr-align:0.01.15"
+        docker:             "kgarimella/lr-align:0.01.17"
```

Comment:
I am a little concerned now about this versioning mechanisms.
Sooner or later, the docker version and the default versions in these WDLs will be out of sync.
But I have no solutions, other than some metaprogramming stuff, which is usually tricky itself...